5/README.md

```diff
@@ -0,0 +1,371 @@
+  | flowers:StaticMenu.FlowersEntry
+  | merchandise:StaticMenu.MerchandiseEntry
+  | plants:StaticMenu.PlantsEntry
+  | prerolls:StaticMenu.PrerollsEntry]
+
+[StaticMenu.ApothecaryEntry
+  | key:string
+  | value:opencannabis.products.Apothecary]
+
+[StaticMenu.CartridgesEntry
+  | key:string
+  | value:opencannabis.products.Cartridge]
+
+[StaticMenu.EdiblesEntry
+  | key:string
+  | value:opencannabis.products.Edible]
+
+[StaticMenu.ExtractsEntry
+  | key:string
+  | value:opencannabis.products.Extract]
+
+[StaticMenu.FlowersEntry
+  | key:string
+  | value:opencannabis.products.Flower]
+
+[StaticMenu.MerchandiseEntry
+  | key:string
+  | value:opencannabis.products.Merchandise]
+
+[StaticMenu.PrerollsEntry
+  | key:string
+  | value:opencannabis.products.Plant]
+
+[StaticMenu.PrerollsEntry
+  | key:string
+  | value:opencannabis.products.Preroll]
+
+[Flag
+  | DRAFT:0
+  | PRIVATE:1
+  | OUT_OF_DATE:2]
+
+[Status
+  | UNPUBLISHED:0
+  | LIVE:1]
+
+{% endnomnoml %}
+
+<a name="products/menu/Menu.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## products/menu/Menu.proto
+
+
+<a name="opencannabis.products.menu.Menu"/>
+
+### Menu
+Holds a full specification for a revision of menu data, segmented into sections, by the categories member products
+are filed in. Categories are enumerated in `menu.Section`.
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| metadata | [Metadata](#opencannabis.products.menu.Metadata) |  | Metadata for the menu. |
+| payload | [SectionedMenu](#opencannabis.products.menu.SectionedMenu) |  | Data payloads attached to this menu. |
+| menu | [StaticMenu](#opencannabis.products.menu.StaticMenu) |  | Specifies a static menu, where each section is specified as a typed map, with keys mapped to products. |
+
+
+<a name="opencannabis.products.menu.MenuProduct"/>
+
+### MenuProduct
+Menu product payload stanza. Specifies a single product as a member of a menu section.
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [opencannabis.base.ProductKey](#opencannabis.base.ProductKey) |  | Section that this data is attached to. |
+| apothecary | [opencannabis.products.Apothecary](#opencannabis.products.Apothecary) |  | Apothecary product. |
+| cartridge | [opencannabis.products.Cartridge](#opencannabis.products.Cartridge) |  | Cartridge product. |
+| edible | [opencannabis.products.Edible](#opencannabis.products.Edible) |  | Edible product. |
+| extract | [opencannabis.products.Extract](#opencannabis.products.Extract) |  | Extract product. |
+| flower | [opencannabis.products.Flower](#opencannabis.products.Flower) |  | Flower product. |
+| merchandise | [opencannabis.products.Merchandise](#opencannabis.products.Merchandise) |  | Merchandise product. |
+| plant | [opencannabis.products.Plant](#opencannabis.products.Plant) |  | Plant product. |
+| preroll | [opencannabis.products.Preroll](#opencannabis.products.Preroll) |  | Preroll product. |
+
```

Comment:
Product descriptions for each section

Reply:
hm, these should definitely go in the core spec. the menus are composed of these objects, but these object themselves shouldn't be specified in an extension.